helm_deploy/values-prod.yaml

```diff
@@ -8,7 +8,6 @@ generic-service:
     AUTH_URI_ROOT: "https://sign-in.hmpps.service.justice.gov.uk/auth"
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://sign-in.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
     COMPLEXITY_OF_NEED_URI: https://complexity-of-need.hmpps.service.justice.gov.uk
-    PRISONS_WITH_OFFENDERS_THAT_HAVE_COMPLEX_NEEDS: "AGI,BZI,DHI,DWI,ESI,EWI,FHI,LNI,NHI,PFI,SDI,STI"
```

Comment:
I can't see anywhere this is still used in the code so probably safe to delete but is it best to keep it in here in case we need to roll back the deployment? Might have misunderstood that point when you made it originally